kitchenowl/lib/l10n/app_en.arb

```diff
@@ -580,6 +587,7 @@
   "shoppingList": "Shopping list",
   "shoppingListDelete": "Delete shopping list",
   "shoppingListDeleteConfirmation": "Are you sure you want to delete {shoppingList}?",
+  "shoppingListContainsEntries": "{entriesCount, plural, =0{The shopping list contains no entrie(s).} =1{The shopping list contains 1 entry.} other{The shopping list contains {entriesCount} entries.}}",
```

Comment:
Can we order these alphabetically? Just move this up two lines. Same for the `@shoppingListContainsEntries`.